src/components/backdrop/backdrop.js

```diff
@@ -66,6 +71,13 @@ angular
         }
       });
 
+      function resize() {
```

Comment:
@crisbeto - my apologies. I am thinking about the dialog not the backdrop. Above is for Backdrop. So disregard my comments plz.